_topic_maps/_topic_map.yml

```diff
@@ -1663,8 +1663,6 @@ Topics:
     File: persistent-storage-csi-migration
   - Name: Detach CSI volumes after non-graceful node shutdown
     File: persistent-storage-csi-vol-detach-non-graceful-shutdown
-  - Name: AliCloud Disk CSI Driver Operator
```

Comment:
Maybe you want to delete the module file? Or is it still included in some other build? 

Also, are you sure there were no xrefs/links to this module apart from the one you remove? Did you do a full text search for IDs in it?

Just some additional checks I would suggest, to be sure everything is in order. If already done, ignore.

Reply:
looks like it shows up here: persistent-storage-csi-sc-manage.adoc

Reply:
LGTM